lib/common/Makefile.am

```diff
@@ -127,6 +127,7 @@ libcrmcommon_test_la_CFLAGS	= $(libcrmcommon_la_CFLAGS) 	\
 # If -fno-builtin is used, -lm also needs to be added.  See the comment at
 # BUILD_PROFILING above.
 libcrmcommon_test_la_LIBADD	= $(libcrmcommon_la_LIBADD) 	\
+				  -lgcov 			\
```

Comment:
We don't require gcc, so you'll have to test availability in configure.ac (or test the --with-coverage option)

Reply:
I've added a patch to give us BUILD_COVERAGE to test against, which means that `./configure --with-coverage` requires gcc but we're not enforcing that at configure time.  I could change it to test for the existence of libgcov, however.  I don't think clang ships with that - at least, I don't see it in our packages.